lib/Delivery.js

```diff
@@ -410,6 +417,14 @@ Delivery.list = function() {
   });
 };
 
+Delivery.get = function(deliveryId) {
+  return API.get(`deliveries/${deliveryId}`).then(function(delivery) {
+    return new Delivery(delivery);
```

Comment:
This doesn't exactly work, because creating a new `Delivery` didn't anticipate receiving, for example, all the `items`. Can you update the Delivery constructor to appropriately set any of the the fields returned by `/deliveries/$ID`:
```
{
    "courier": {
        "location": {
            "latitude": 40.7619629893,
            "longitude": -74.0014480227,
            "bearing": 33
        },
        "name": "Rob",
        "phone": "+18622564699",
        "picture_url": "https://d297l2q4lq2ras.cloudfront.net/nomad/2014/10/16/18/479x479_id_dba13493-27db-4d39-a322-8cb5eca67b54.jpeg",
        "vehicle": {
            "license_plate": "RUSHNYC",
            "make": "Acura",
            "model": "ZDX",
            "picture_url": null
        }
    },
    "created_at": 1441147296,
    "currency_code": "USD",
    "delivery_id": "8b58bc58-7352-4278-b569-b5d24d8e3f76",
    "dropoff": {
        "contact": {
            "company_name": "Gizmo Shop",
            "email": "contact@uber.com",
            "first_name": "Calvin",
            "last_name": "Lee",
            "phone": {
                "number": "+14081234567",
                "sms_enabled": false
            },
            "send_email_notifications": true,
            "send_sms_notifications": true
        },
        "eta": 30,
        "location": {
            "address": "530 W 113th Street",
            "address_2": "Floor 2",
            "city": "New York",
            "country": "US",
            "postal_code": "10025",
            "state": "NY"
        },
        "signature_required": false,
        "special_instructions": null
    },
    "fee": 5.0,
    "items": [
        {
            "height": 5.0,
            "is_fragile": false,
            "length": 14.5,
            "price": 1.0,
            "quantity": 1,
            "title": "Shoes",
            "weight": 2.0,
            "width": 7.0
        },
        {
            "height": 5.0,
            "is_fragile": false,
            "length": 25.0,
            "quantity": 1,
            "title": "Guitar",
            "weight": 10.0,
            "width": 12.0
        }
    ],
    "order_reference_id": "SDA124KA",
    "pickup": {
        "contact": {
            "company_name": "Gizmo Shop",
            "email": "contact@uber.com",
            "first_name": "Calvin",
            "last_name": "Lee",
            "phone": {
                "number": "+14081234567",
                "sms_enabled": false
            },
            "send_email_notifications": true,
            "send_sms_notifications": true
        },
        "eta": 4,
        "location": {
            "address": "636 W 28th Street",
            "address_2": "Floor 2",
            "city": "New York",
            "country": "US",
            "postal_code": "10001",
            "state": "NY"
        },
        "special_instructions": "Go to pickup counter in back of shop."
    },
    "quote_id": "KEBiNmI4MWQ0NDIzNjUyNjE1ZmM5YzlkNDQ5NDA4MzhlNTg5MWZlNzQ5YTNmZTRkYzQxZTgxMzc4YjlkZjU0Yjc2Fbamsd4KFeavsd4KFYD1sgwcFdD/0oQDFYfw48EFABwVyoCThQMVp/qvwQUAGANVU0QA",
    "status": "en_route_to_pickup",
    "tracking_url": "https://trip.uber.com/v2/share/-JazZXXuBl",
    "batch": {
      "batch_id": "963233d3-e8ad-4ed9-aae7-95446ffee22f",
      "count": 2,
      "deliveries": ["8b58bc58-7352-4278-b569-b5d24d8e3f76", "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"]
    }
}
```